lib/workers/repository/onboarding/branch/config.ts

```diff
@@ -1,9 +1,35 @@
 import { RenovateConfig } from '../../../../config';
+import { getPreset } from '../../../../config/presets';
+import { PRESET_DEP_NOT_FOUND } from '../../../../config/presets/util';
 import { logger } from '../../../../logger';
 import { clone } from '../../../../util/clone';
 
-export function getOnboardingConfig(config: RenovateConfig): string {
-  const onboardingConfig = clone(config.onboardingConfig);
+export async function getOnboardingConfig(
+  config: RenovateConfig
+): Promise<string> {
+  let onboardingConfig = clone(config.onboardingConfig);
+
+  let organizationConfigRepoExists = false;
+  const organizationConfigPresetName = `local>${config.owner}/renovate-config`;
+
+  try {
+    if (config.owner) {
+      await getPreset(organizationConfigPresetName, config);
+      organizationConfigRepoExists = true;
+    }
+  } catch (err) {
+    if (err.message !== PRESET_DEP_NOT_FOUND) {
+      throw err;
+    }
+    // Organization preset did not exist
+  }
+  if (organizationConfigRepoExists) {
+    onboardingConfig = {
```

Comment:
I have doubts that we want to completely disregard `config.onboardingConfig` here. Are we sure about that?

Reply:
In the hosted app case, the default config is just set to extending the base, right? I think that someone making the `renovate-config` preset for their org will want to be overriding it completely

In the case where the bot is being run self-hosted, my assumption would be that the people running the bot would be wanting to set a default for everything their bot touches, but still let an individual organisation/group etc set finer-grained defaults. If there's a conflict between those two people, they can presumably resolve it themselves.

I can't see any way we could merge it - part of the reason someone might want a different preset is they don't want config:base? And they can after all set whatever settings they want within that preset?

Reply:
Good point. I agree with your approach.